doc/cs_registers.rst

```diff
@@ -65,19 +65,20 @@ CSR Address: ``0x300``
 
 Reset Value: ``0x0000_1800``
 
-+-------+-----+------------------------------------------------------------------+
-| Bit#  | R/W | Description                                                      |
-+-------+-----+------------------------------------------------------------------+
-| 12:11 | R   | **MPP:** Statically 2’b11 and cannot be altered (read-only).     |
-+-------+-----+------------------------------------------------------------------+
-| 7     | RW  | **Previous Interrupt Enable:** When an exception is encountered, |
-|       |     | MPIE will be set to IE. When the mret instruction is executed,   |
-|       |     | the value of MPIE will be stored to IE.                          |
-+-------+-----+------------------------------------------------------------------+
-| 3     | RW  | **Interrupt Enable:** If you want to enable interrupt handling   |
-|       |     | in your exception handler, set the Interrupt Enable to 1’b1      |
-|       |     | inside your handler code.                                        |
-+-------+-----+------------------------------------------------------------------+
++-------+-----+---------------------------------------------------------------------------------+
+| Bit#  | R/W | Description                                                                     |
++-------+-----+---------------------------------------------------------------------------------+
+| 12:11 | R   | **MPP:** Statically 2’b11 and cannot be altered (read-only).                    |
++-------+-----+---------------------------------------------------------------------------------+
+| 7     | RW  | **Previous Interrupt Enable (MPIE)**, i.e., before entering exception handling. |
++-------+-----+---------------------------------------------------------------------------------+
+| 3     | RW  | **Interrupt Enable (MIE):** If set to 1'b1, interrupts are globally enabled.    |
++-------+-----+---------------------------------------------------------------------------------+
+
+When an exception is encountered, MPIE will be set to MIE.
+When the ``mret`` instruction is executed, the value of MPIE will be stored back to IE.
+
+If you want to enable interrupt handling in your exception handler, set MIE to 1’b1 inside your handler code.
```

Comment:
nit: looks like you got a wrong apostrophe/single quotation mark in 1'b1?

Reply:
Amazing, that you noted that!